lib/matplotlib/cbook.py

```diff
@@ -1088,6 +1088,45 @@ def _combine_masks(*args):
     return margs
 
 
+def broadcast_with_masks(*args, compress=False):
```

Comment:
```suggestion
def _broadcast_with_masks(*args, compress=False):
```
As with the `combine_masks()` above, this is not user facing and we should strive to keep our internals private.